dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/newsession/NewSessionPopup.java

```diff
@@ -24,13 +25,18 @@
 @Singleton
 public class NewSessionPopup {
     private final ResourceBundle resourceBundle;
+    private final Popup popup;
 
     @Inject
-    public NewSessionPopup(final ResourceBundle resourceBundle) {
+    public NewSessionPopup(
+            final ResourceBundle resourceBundle, final Ds3Common ds3Common,
```

Comment:
To be consistent, put `ds3Common` on it's own line.

Reply:
actually, looking at this a second time, I don't see where `ds3Common` is being used.